src/is-where.js

```diff
@@ -1,28 +1,72 @@
 const entries = require('object.entries');
 
+const ATTRIBUTE_PRESENT = {exists: true};
+
 const _isWhere = (where, target) => {
```

Comment:
There is enough diverse things converted in here that I think a doc block before the function would be super nice that at minimum has a bullet list of the conversions with short descriptions/syntax.